deploy/olm-catalog/service-assurance-operator/0.1.0/service-assurance-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -16,13 +18,25 @@ metadata:
         }
       ]
     capabilities: Basic Install
-  name: service-assurance-operator.v0.1.0
-  namespace: sa-telemetry
+    categories: Monitoring
+    description: Service Assurance Framework. Umbrella Operator for instantiating the required dependencies and configuration of various components to build a Service Assurance platform for telco grade monitoring.
+    certified: "false"
+    createdAt: 2019-11-04T08:00:00Z
+    support: Red Hat, Inc.
+    repository: https://github.com/redhat-service-assurance/service-assurance-operator
+    containerImage: quay.io/redhat-service-assurance/service-assurance-operator.0.1.0
```

Comment:
I'd like to have a real-time conversation with you sometime this week about upstream and downstream for this stuff.

Reply:
Sounds like I'm getting called to the principals office.